semantic/src/site/globals/site.overrides

```diff
@@ -2818,6 +2820,7 @@ h6:first-child {
   }
   #overview .left-item {
     min-width: unset;
+    padding-right: 5px;
```

Comment:
Fix small gap in labels as they wrap.

![label_gap_finish](https://github.com/msupply-foundation/conforma-web-app/assets/17306152/8a3a5fc3-2080-416e-abf1-1ba524634c19)